code/modules/hallucinations/effects/moderate.dm

```diff
@@ -476,3 +476,34 @@
   */
 /obj/effect/hallucination/delusion/proc/get_image(mob/living/carbon/human/H)
 	return image('icons/mob/animal.dmi', H, pick("black_bear", "brown_bear", "corgi", "cow", "deer", "goat", "goose", "pig", "blank-body"))
+
+// Doppelganger hallucination
+// Spawns a copy of the player that briefly follows them around
+/obj/effect/hallucination/doppelganger
+	duration = 10 SECONDS
+	var/obj/effect/hallucination/chaser/you/fake_you = null
```

Comment:
Please make sure this is neatly removed on Destroy()

Reply:
Isn't this already being done on /hallucination/ or am I dumb for thinking that?

Reply:
Would qdel-ing the var in Destroy() suffice?

Reply:
QDEL_NULLing it would be preferred

Reply:
Should be in now